src/utils/configfile.cc

```diff
@@ -88,7 +88,9 @@ bool ConfigFile::Init(OptionsParser* options) {
   // loading the default configuration file.
   if (filename == "") return true;
 
-  filename = CommandLine::BinaryDirectory() + "/" + filename;
+  if (filename.at(0) != '/') {
```

Comment:
This fix will work on Unix systems, but seems to leave Windows (who likes to prefix paths with `C:\`, `D:`, etc) users in the cold.

Reply:
I have added a preprocessor directive to check for windows, then check for a colon in that path. I don't have a way to whether the windows section is robust though.